### ISSUE TITLE
[Design, FE] 10-10EZ Short-Form: Add unique name for Insurance Information edit buttons > [Link and button labels] Button and link language isn't descriptive and/or their purpose isn't clear. (08.03.3)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#40490]
#### VFS team name
10-10 Team

#### VFS product name
Health Care Application 10-10EZ

#### VFS feature name
10-10EZ Short Form (Auto-qualifiers)

#### Point of Contact/Reviewers
Brian DeConinck (@briandeconinck) - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Button and link language isn't descriptive and/or their purpose isn't clear.

### Issue Details
On Step 2 of 3 [disability rating >50%]: Insurance Information, when a user has health insurance coverage and is adding multiple insurance policies, an Edit button is paired with each policy. Each of these Edit buttons have identical accessible names, "Edit." For screen reader users tabbing through the buttons, it's difficult to identify which policy will be edited by following which button.

### Link, screenshot or steps to recreate
1. Log into the RI as user 228.
2. Complete the form up to Step 2 of 3.
3. For "Do you have health insurance coverage?" answer Yes.
4. Enter information for an insurance policy.
5. Click "Add another insurance policy."
6. Enter information for another insurance policy.
7. Click "Add another insurance policy."
8. Activate a screen reader (recommend NVDA on Windows or VoiceOver on Mac for testing).
9. Press [tab] or [shift] + [tab] to move forward and backward through the interactive elements on the page.
10. Note that the edit buttons for each insurance policy are announced only as "Edit."

### VA.gov Experience Standard
[Category Number 08, Issue Number 03](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.4_A

---

### Platform Recommendation
Each edit button should have a unique accessible name. This can be accomplished either by changing the button text (`<button>Edit [name of policy]</button>`) or by providing a unique aria-label (`<button aria-label="Edit [name of policy]">Edit</button>`).

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by hdjustice** [comment]
@briandeconinck 
This functionality is in production now.  The team will need to do some investigation, as they believe this is a platform issue and not within our control.

Do you still feel this should be a launch blocker to the Short Form feature, even though this is in production today?

**Comment by lzapson** [comment]
This is a forms issues, we cannot change the aria label in the application code when you can add multiple items of the same kind, like insurance companies. You can find the platform code that is rendering the edit button on the Insurance information page, [here](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/forms-system/src/js/fields/ArrayField.jsx#L339).

This is also happening on the Review Application page, when a user edits Insurance Information, which is also a forms issue. You can find the platform code that is rendering the Insurance information edit button on the Review Application page, [here](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/forms-system/src/js/review/ObjectField.jsx#L170).

@shiragoodman @hdjustice  please assign to the appropriate team.

**Comment by hdjustice** [comment]
>This is a forms issues, we cannot change the aria label in the application code when you can add multiple items of the same kind, like insurance companies. You can find the platform code that is rendering the edit button on the Insurance information page, here.
>
>This is also happening on the Review Application page, when a user edits Insurance Information, which is also a forms issue. You can find the platform code that is rendering the Insurance information edit button on the Review Application page, here.
>
>@shiragoodman @hdjustice please assign to the appropriate team.

@shiragoodman - hey there!  How should we handle this ticket?  
Per Lisa's comment above, this issue stems from the forms component, and the aria label cannot be changed.  
Thanks!

**Comment by shiragoodman** [comment]
@hdjustice removing your team label.

@caw310 @k80bowman reassigning to Design System Team.

**Comment by micahchiang** [comment]
The original issue mentioned that screen readers only announce "edit", but it seems like things are working as expected? Each of these items have an aria-label value of `Edit insurance policy`, which is correctly read out by VoiceOver. I was unable to test on JAWS or NVDA though. 

<img width="1680" alt="Screen Shot 2022-05-26 at 9 05 38 AM" src="https://user-images.githubusercontent.com/15097156/170504580-5a775430-27bb-4215-92c2-bf0d018c3d1c.png">

**Comment by shiragoodman** [comment]
@briandeconinck please see Micah's comments above.

**Comment by taylorkaren** [comment]
Transferred to https://app.zenhub.com/workspaces/forms-library---platform-spike-team-61b0ae1f2cd3c30014e8a5b0/issues/department-of-veterans-affairs/va-forms-system-core/357.

**Comment by hdjustice** [comment]
@briandeconinck - This ticket moved around a bit since we needed some help from the design system team.  It has come back to us to address, however I still have the same comment:

> The 10-10EZ short form skips over this question, and does not appear during this process. We agree that this does need to be addressed sooner rather than later, however this is in production now and we feel it should not be a launch blocker to functionality that does not display it.
>Can you give us your thoughts on whether this has to be a launch blocker to the Short Form release? Or can we address it separately as a fast follow?

cc: @markfallows

**Comment by hdjustice** [comment]
Thank you @briandeconinck  - We will look into this and get it scheduled in the next few sprints.

cc: @markfallows
